core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -126,6 +128,11 @@ private void writeComparisons(JPlagResult result) {
         submissionNameToNameToComparisonFileName = comparisonReportWriter.writeComparisonReports(result);
     }
 
+    private void writeBaseCodeReport(JPlagResult result) {
+        BaseCodeReportWriter baseCodeReportWriter = new BaseCodeReportWriter(submissionToIdFunction, this.resultWriter);
```

Comment:
This would need some bigger changes to the file handling of the report viewer, since it currently expects all files that should be included in a report to be included and throws an error otherwise. If you prefer to have the check tho, I could of course change it

Reply:
Ah, in this code we don't need the check. I was under the assumption no basecode files are written anyways. Let's leave it as is.